A few fixes (#1)
Zarkonnen/three-new-starters: data/ships.txt

> ship "SpaceJumper" 

Author: ```suggestion
ship "Space Jumper"
```
Will delete the space on the end, but two words might work better?

> ship "SpaceJumper" 

Author: ```suggestion
ship "SpaceJumper"
```

		"LP036a Battery Pack"
		
	engine 0 19
	explode "tiny explosion" 10

> 	description `The StarGate Inc. SpaceJumper is the smallest craft capable of fitting a hyperdrive. It's sold in kit form and advertised to "everyone who ever wanted their own spaceship". Most buyers who manage to assemble the ship make one terrifying trip to Alpha Centauri and back, and then sell the thing for scrap.`

Author: ```suggestion
	description `The StarGate Inc. Space Jumper is the smallest craft capable of fitting a hyperdrive. It's sold in kit form and advertised to "everyone who ever wanted their own spaceship". Most buyers who manage to assemble the ship make one terrifying trip to Alpha Centauri and back, and then sell the thing for scrap.`
```

Author: And about the engine: I’m pretty sure they are under by default as well.

Author: Actually, it would appear I’m wrong according to [line 2613 in Engine.cpp](https://github.com/endless-sky/endless-sky/blob/51de725e6eac78abf413e8025cc7e8efc3c91c7b/source/Engine.cpp#L2613), based on my extremely limited C++ knowledge. 
The question is, should it be under? Based on all the other ships no (the only use of `under` in `human/ships.txt` is for fighter and drone bays), but then again it’s your choice.

Reviewer: I eyeballed it and without under looked fine. The smooshed together "SpaceJumper" instead of "Space Jumper" is a deliberate obnoxiousness. All other suggested changes look good.

		"nGVF-BB Fuel Cell"
		
	engine 0 39
		under
	gun -6 -39
		under
	gun 6 -39

> 		under

Author: Guns are under by default.

		mass 100
		bunks 11
		"cargo space" 10

> 		drag 1.75

Author: A bit slower is probably better.

	engine 0 63
	explode "small explosion" 5
	explode "small explosion" 10

> 	description `A small passenger craft popular in the Core systems for ferrying around middle-class people in moderate discomfort. Syndicated Shipyards advertises that travelling in a Starbus hearkens back to the first golden age of air travel on Earth.`

Author: Core: in-game it should be capitalized (ref. `FW Katya 2`’s `on offer` section)
And the player will know what `whatever that may mean` will mean, and so should `<first>` `<last>`.
